### DISCUSSION TITLE
Z Up | Work through Viability

### DISCUSSION THREAD

**Discussion by jjburton** [jjburton/cgmTools#185] [Ideas]
Discussion for issues pertaining to getting z up working properly with MRS build. For any interested in this, it would be good start to:
- if you're interested do the intro to blocks section of the workshop with a z up world
- See what breaks. 
- Post any notes here
- We can start a branch and start working through it.

**Comment by benngarnish** [comment]
A note on this that I’ve found before. 
making sure your maya settings are also set for maya to read as Z being the world up axis.
